src/Rule/LatteTemplatesRule.php

```diff
@@ -98,6 +98,10 @@ public function processNode(Node $collectedDataNode, Scope $scope): array
             }
         }
 
+        if (count($errors) > 1000) {
+            $errors[] = RuleErrorBuilder::message('Too many errors in latte.')->build();
+        }
```

Comment:
Here should be some break / return

Reply:
This is after analysis is done. I break analysis in different palce. BUt to prevent adding this error multiple times (for every resolver and resolved node) I add it at the end or analysis